File: 1-Iniciante/1066-Pares-Impares-Positivos-e-Negativos/1066.py
a = []
for i in range(5):
    a.append(int(input()))
b = {"par" : 0, "impar" : 0, "negativo" : 0, "positivo" : 0}
for i in a:
    if i % 2 == 0:
        b.update({"par" : b["par"] + 1})
    else:
        b.update({"impar" : b["impar"] + 1})
    if i > 0:
        b.update({"positivo" : b["positivo"] + 1})
    elif i < 0:
        b.update({"negativo" : b["negativo"] + 1})
print (b["par"], "valor(es) par(es)")
print (b["impar"], "valor(es) impar(es)")
print (b["positivo"], "valor(es) positivo(s)")
print (b["negativo"], "valor(es) negativo(s)")

File: 1-Iniciante/1066-Pares-Impares-Positivos-e-Negativos/1066.c
#include <stdio.h>

int main() {
    int nums[4] = {0, 0, 0, 0}, n;
    for (int i = 0; i <= 4; i++) {
        scanf("%d", &n);
        if (n % 2 == 0) {
            nums[0] += 1;
        }
        else {
            nums[1] += 1;
        }
        if (n > 0) {
            nums[2] += 1;
        }
        else if (n < 0) {
            nums[3] += 1;
        }
    }
    printf ("%d valor(es) par(es)\n", nums[0]);
    printf ("%d valor(es) impar(es)\n", nums[1]);
    printf ("%d valor(es) positivo(s)\n", nums[2]);
    printf ("%d valor(es) negativos(s)\n", nums[3]);

    return 0;
}